webpack.config.js

```diff
@@ -103,16 +104,41 @@ if (process.env.NODE_ENV === 'production' && process.env.IS_PUBLISHING) {
   );
 }
 
+const entry = {
+  index: [path.join(__dirname, 'src/index.ts')], // React components
+  utilities: [path.join(__dirname, 'src/styles/utilities.scss')], // Utilities CSS only.
+  fonts: [path.join(__dirname, 'src/styles/fonts.scss')], // Fonts CSS only.
+  variables: [path.join(__dirname, 'src/styles/variables/index.scss')], // Variables CSS only.
+  reset: [path.join(__dirname, 'src/styles/reset.scss')], // CSS Reset only.
+  Box: [path.join(__dirname, 'src', 'components', 'Box', 'Box.tsx')],
+};
+
+fs.readdir(path.join(__dirname, 'src', 'components'), (err, files) => {
```

Comment:
I couldn't actually get this working, it was only using the stuff in entry already